### DISCUSSION TITLE
Enhancement: Loading screen

### DISCUSSION THREAD

**Discussion by AngelHQ** [FunkinCrew/Funkin#1614] [Ideas]
I think a loading screen shoiuld help fixing the performance issues, and maybe some bugs related to it. It'd be putting it between songs from the same week and do technical stuff like refreshing the buffer and waiting for the song to load before starting.
It could also wait for a prompt from the player to press any button and start reproducing the next song.

**Comment by Windows1Door2Daniel** [comment]
![Untitled](https://user-images.githubusercontent.com/81389527/114502362-81f18200-9be0-11eb-9214-c37323da0172.png)

**Comment by AngelHQ** [comment]
That's a nice loading screen Salladk-but-no-mo-flaged! There could be one for each foe Bf encounters. Don't be afraid to move that image around for anyone who couldbe interested :)

**Comment by Windows1Door2Daniel** [comment]
> That's a nice loading screen Salladk-but-no-mo-flaged! There could be one for each foe Bf encounters. Don't be afraid to move that image around for anyone who couldbe interested :)

well i actually think i should change each icon for each rival but i got lazy so i only did Daddy dearest V.S. Boyfriend
